int16 modeNumber
int16 status
